inertial_frame_id: world
waypoints:
  -
    point: [0, 0, -18]
    max_forward_speed: 0.5
    heading: 0
    use_fixed_heading: False
  -
    point: [0, -3, -18]
    max_forward_speed: 0.5
    heading: 0
    use_fixed_heading: False
  -
    point: [0, -6, -18]
    max_forward_speed: 0.3
    heading: 0
    use_fixed_heading: False
 